### PR TITLE
Fixed phpdoc

### PR DESCRIPTION
**Pull request by core23**

### CODE REVIEW

**Comment by OskarStark** [comment]
Thank you @core23

File: Spread/AdminSpread.php [hunk 1]
```diff
@@ -19,18 +19,30 @@
 
 class AdminSpread implements SpreadInterface
 {
+    /**
+     * @var array
```
Comment (Soullivaneuh) [inline Spread/AdminSpread.php]: `string[]`

File: Spread/AdminSpread.php [hunk 2]
```diff
@@ -19,18 +19,30 @@
...
+     */
     protected $supportedVerbs = array(
         'sonata.admin.create',
         'sonata.admin.update',
         'sonata.admin.delete',
     );
 
+    /**
+     * @var array
```
Comment (Soullivaneuh) [inline Spread/AdminSpread.php]: `string[]`

**Comment by Soullivaneuh** [comment]
@OskarStark Some phpdoc was not accurate.

@core23 Could you please make another PR for that? Thanks!

**Comment by OskarStark** [comment]
@Soullivaneuh sorry for that, but is this really "wrong"? or is ``string[]`` just more perfectly?

**Comment by Soullivaneuh** [comment]
`array` is correct but `string[]` is more accurate. ;-)

**Comment by OskarStark** [comment]
good to know, thank you :smile: 

:+1:

**Comment by core23** [comment]
@rande @Soullivaneuh should we always use the type defining notation for arrays or only at special occurrences? And how do we document doctrine ``ArrayList``s and ``Collection``s?

**Comment by Soullivaneuh** [comment]
> And how do we document doctrine ArrayLists and Collections?

```php
MyEntity[]|ArrayCollection
```

For example.

**Pull request merged**